src/cron/personalizedDigest.ts

```diff
@@ -14,11 +15,18 @@ const cron: Cron = {
       });
 
     const personalizedDigestStream = await personalizedDigestQuery.stream();
+    const notifyQueueConcurrency = 1000;
+    const notifyQueue = fastq.promise(
+      async (personalizedDigest: UserPersonalizedDigest) => {
+        await notifyGeneratePersonalizedDigest(logger, personalizedDigest);
+      },
```

Comment:
```suggestion
      (personalizedDigest: UserPersonalizedDigest) => notifyGeneratePersonalizedDigest(logger, personalizedDigest),
```

Reply:
@idoshamun for some reason with above changes tests timed out, not sure if it's a jest thing, fastq thing https://app.circleci.com/pipelines/github/dailydotdev/daily-api/5895/workflows/db8d60e6-58fc-422b-ac89-dc8f4f0bc8fa/jobs/23036 cc: @rebelchris any clue?

Reverted below and tests passed again.